File: 1146A_Love.cpp
#include <iostream>
using namespace std;

int main() {
    string s;
    cin >> s;
    int cont = 0;
    int otro = 0;

    for (char i : s) {
        if (i == 'a') {
            cont += 1;
        } else {
            otro += 1;
        }
    }
    while (otro >= cont) {
        otro -= 1;
    }
    cout << cont + otro << endl;

    return 0;
}

/*
Entrada:
 - Una línea: cadena s (contiene al menos una 'a')
Salida:
 - Un entero: longitud máxima de la cadena resultante donde 'a' > mitad
Ejemplo:
Entrada:
aaabbb
Salida:
4
*/

File: 1146A_Love.py
"""
Entrada:
 - Una línea: cadena s (contiene al menos una 'a')
Salida:
 - Longitud máxima de una subcadena (obtenida borrando caracteres) tal que el número de 'a' sea > la mitad
Ejemplo:
Entrada:
aaabbb
Salida:
4
"""

s = input()
conta = 0
otro = 0

for i in s:
    if i == 'a':
        conta += 1
    else:
        otro += 1

while otro >= conta:
    otro -=1

print(conta+otro)